mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/CassandraMailboxManagerTest.java

```diff
@@ -875,10 +863,7 @@ private CassandraThreadLookupDAO threadLookupDAO(CassandraCluster cassandraClust
         }
 
         private Mono<Void> saveThreadData(Username username, Set<MimeMessageId> mimeMessageIds, MessageId messageId, ThreadId threadId, Optional<Subject> baseSubject) {
-            return threadDAO(cassandra.getCassandraCluster())
-                .insertSome(username, mimeMessageIds, messageId, threadId, baseSubject)
-                .then(threadLookupDAO(cassandra.getCassandraCluster())
-                    .insert(messageId, username, mimeMessageIds));
+            return Mono.empty();
```

Comment:
I guess should remember to put those back and fix it correct?

Reply:
Yes